CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.1.z]
 
+## [1.1.13] - 2024-06-11
+
+> There is no certainty in the world. This is the only certainty I have.
+
+### Important Notes
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Fixed
+
+ * Support go 1.22.4+. (#4313)
+ * runc list: fix race with runc delete. (#4231)
+ * Fix set nofile rlimit error. (#4277, #4299)
+ * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4284)
+ * Fix codespell warnings. (#4300)
+ * Fix a debug msg for user ns in nsexec. (#4315)
+ * script/*: fix gpg usage wrt keyboxd. (#4316)
+
+### Changed
+
+ * CI fixes and misc backports. (#4241)
+ * Silence security false positives from golang/net. (#4244)
+ * Vagrantfile.fedora: bump Fedora to 39. (#4261)
+ * libcontainer: force apps to think fips is enabled/disabled for testing. (#4257)
```

Comment:
The commit message for this change didn't make sense, this is a better way of phrasing what the change actually was:

```suggestion
 * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4257)
```